src/cpp/fastdds/builtin/type_lookup_service/TypeLookupRequestListener.cpp

```diff
@@ -17,17 +17,17 @@
  *
  */
 
+#include <fastdds/builtin/type_lookup_service/TypeLookupRequestListener.hpp>
 
-#include <fastrtps/types/TypeObjectFactory.h>
+#include <utility>
 
-#include <fastrtps/rtps/reader/StatefulReader.h>
 #include <fastrtps/rtps/history/ReaderHistory.h>
-#include <fastdds/dds/log/Log.hpp>
+#include <fastrtps/rtps/reader/StatefulReader.h>
+#include <fastrtps/types/TypeObjectFactory.h>
 
-#include <utility>
+#include <fastdds/dds/log/Log.hpp>
```

Comment:
This public header should be in the block above.

Reply:
[f288915](https://github.com/eProsima/Fast-DDS/pull/4053/commits/f288915aa343eabd6476d3875ea51d994ebe899d)